operations/helm/charts/alloy/templates/networkpolicy.yaml

```diff
@@ -0,0 +1,25 @@
+{{- if and .Values.networkPolicy.enabled (eq .Values.networkPolicy.flavor "kubernetes") -}}
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+metadata:
+  name: {{ include "alloy.fullname" . }}
+  namespace: {{ include "alloy.namespace" . }}
+  labels:
+    {{- include "alloy.labels" . | nindent 4 }}
+    app.kubernetes.io/component: networking
+spec:
+  podSelector:
+    matchLabels:
+      {{- include "alloy.selectorLabels" . | nindent 6 }}
+  policyTypes:
+    - Ingress
+    - Egress
```

Comment:
Should we not list only the ones that are defined? E.g. either ingress or egress, or both?

Reply:
Good point.  Will fix.

Reply:
I moved `policyTypes` to the values file